fontc/src/args.rs

```diff
@@ -21,12 +21,15 @@ pub struct Args {
     #[arg(long, default_value = "false")]
     pub emit_debug: bool,
 
-    /// Whether to Try Hard(tm) to match fontmake (Python) behavior in cases where there are other options.
-    ///
-    /// See <https://github.com/googlefonts/fontmake-rs/pull/123> for an example of
-    /// where this matters.
+    /// In cases where a source glyph uses a mixture of components and contours, convert
+    /// all the components to contours.
     #[arg(long, default_value = "true")]
-    pub match_legacy: bool,
+    pub prefer_simple_glyphs: bool,
+
+    /// Eliminate component references to other glyphs using components, emitting only
+    /// component references to simple (contour) glyphs.
+    #[arg(long, default_value = "true")]
+    pub flatten_components: bool,
```

Comment:
I would very much prefer to stick to vanilla fontmake default behavior which doesn _not_ flatten components by default, see https://github.com/googlefonts/fontmake-rs/issues/250

Reply:
Per IM I agree, will so update.